src/renderer.rs

```diff
@@ -365,6 +365,25 @@ fn entry_row(
     }
 }
 
+fn bytes_to_size(bytes: u64) -> String {
+    const UNITS: [&str; 7] = ["B", "KB", "MB", "GB", "TB", "PB", "EB"];
```

Comment:
Isn't this technically incorrect as these really appear to be the SI units instead since you use 1024 everywhere? I'm definitely ok with using SI units but we should call them correctly, too.